### DISCUSSION TITLE
Renovate not enabled for repo even though it's turned on in settings

### DISCUSSION THREAD

**Discussion by Vijay-AlphawatchAI** [renovatebot/renovate#32905] [Request Help]
### How are you running Renovate?

A Mend.io-hosted app

### If you're self-hosting Renovate, tell us which platform (GitHub, GitLab, etc) and which version of Renovate.

Github Renovate App

### Please tell us more about your question or problem

I turned on the Renovate app for a repo (called search-engine) owned by my company's Github organization. When I go to the Renovate app settings, it both says that there's no repos under the organization that are enabled but also that it's installed in the search-engine repo. Any idea how I can debug what the status of Renovate is on this repo? 

<img width="564" alt="image" src="https://github.com/user-attachments/assets/4a6b1816-7882-4185-bb0a-5a7a7c1b89e4">

Also, I'm not an owner of this organization. What level of permissions do I need to be able to manage Renovate for our repos? Thanks for your help!

### Logs (if relevant)

<details><summary>Logs</summary>

```

Replace this text with your logs, between the starting and ending triple backticks

```

</details>

**Comment by rarkins** [comment]
Please show the other screenshot of it "not working"

**Reply by Vijay-AlphawatchAI** [reply]
Sorry when I say it's "not working" I mean that I haven't gotten the welcome PR so I don't know the current status past that. I enabled this well over two weeks ago so it's not a timing thing. Is there a way I can see why Renovate is failing to process my repo?

**Reply by rarkins** [reply]
Log into https://developer.mend.io and see if it's visible. It's possible it was installed in "silent" mode so no onboarding would be expected
